packages/package.go

```diff
@@ -457,6 +458,7 @@ func (p *Package) setBasePolicyTemplates() {
 			Categories:      t.Categories,
 			Icons:           t.Icons,
 			DeploymentModes: t.DeploymentModes,
+			BaseDataStreams: t.DataStreams,
```

Comment:
Policy templates that don't declare their data streams use all data streams in the package, but their list of data streams will be empty here.

Is this ok for clients? Should we fill the data streams list with the list of data streams in the package in this case?

They could also obtain the full list of data streams from `data_streams`, but it doesn't currently contain its names, it would need to be guessed from the data sets. Or should we add the names of the data streams to `data_streams`?

Reply:
> Policy templates that don't declare their data streams use all data streams in the package, but their list of data streams will be empty here.
Is this ok for clients? Should we fill the data streams list with the list of data streams in the package in this case?

I think an empty array in that case is acceptable, and should be handled gracefully by the clients